<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE Procedure [] >
  <Procedure
      designNotes="Undefined"
      isTPL="false"
      language="EcmaScript"
      name="handleUnexpectedException"
      nested="false"
      version="4.1.0">
    <ReferenceParameters />
    <ObjectField
        name="exception">
      <TypeDefinitionReference
          name="Exception"
          nested="false" />
    </ObjectField>
    <ProcedureLocals />
    <Verbatim
        fieldName="text">
<![CDATA[  getAssertor().assert(false, "UnitTest failed due to an uncaught "+exception.getTypeName() +": "+exception.getErrorMessage());
]]>
    </Verbatim>
  </Procedure>
